charts/actions-runner-controller/values.yaml

```
@@ -63,6 +57,16 @@ service:
   type: ClusterIP
   port: 443
 
+metrics:
+  serviceMonitor: false
+  port: metrics-port
+  portNumber: 8443
```

```suggestion
  port: 8443
```

I think we can just use the fixed name and use `port` for the port number instead, so that it is more consistent with our own other port-related settings like `service.port` and common charts like https://github.com/helm/charts/blob/7e45e678e39b88590fe877f159516f85f3fd3f38/stable/fluent-bit/values.yaml#L36

---

done